spec/system/comments/user_edits_a_comment_spec.rb

```diff
@@ -10,6 +10,7 @@
            user: user,
            body_markdown: Faker::Lorem.paragraph)
   end
+  let(:iso8601_datetime_regexp) { /^((\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})Z)$/ }
```

Comment:
thanks @jacobherrington for this :D

Reply:
![image](https://user-images.githubusercontent.com/3534427/109510340-c9410a80-7a99-11eb-83a4-e08d14b6cb4e.png)

Just in case someone wants to see what the [regex](https://regexper.com/#%2F%5E%28%28%5Cd%7B4%7D%29-%28%5Cd%7B2%7D%29-%28%5Cd%7B2%7D%29T%28%5Cd%7B2%7D%29%3A%28%5Cd%7B2%7D%29%3A%28%5Cd%7B2%7D%29Z%29%24%2F) does 😁

Reply:
Thank you @Link2Twenty :D